service/controller/chart/resource/status/create.go

```diff
@@ -100,9 +101,27 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	return nil
 }
 
+func (r *Resource) getAuthToken(ctx context.Context) (string, error) {
+	secret, err := r.k8sClient.CoreV1().Secrets(namespace).Get(ctx, authTokenName, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		// it is certainly control plane apps or auth secret is not created yet.
+		r.logger.LogCtx(ctx, "level", "debug", "message", "no auth token secret yet in cluster")
```

Comment:
Logging if we can't find a secret would be nice, just to save our mental later time.